client/blocks/signup-form/passwordless.jsx

```diff
@@ -297,7 +300,26 @@ class PasswordlessSignupForm extends Component {
 	}
 
 	getLabelText() {
-		return this.props.labelText ?? this.props.translate( 'Enter your email address' );
+		return (
+			<div className="signup-form__passwordless-email-label">
+				<FormLabel>
+					{ this.props.labelText || this.props.translate( 'Enter your email address' ) }
+				</FormLabel>
+			</div>
+		);
+	}
+
+	getFormButtonAndToS() {
+		const isPasswordlessInviteForm = this.props.inviteFormTitle;
+		return isPasswordlessInviteForm ? (
+			<>
+				{ this.formFooter() } { this.props.renderTerms?.() }
```

Comment:
Other pages have different order, [example](https://wordpress.com/start/account/user-social?variationName=link-in-bio&redirect_to=%2Fsetup%2Flink-in-bio%2Fpatterns&pageTitle=Link+in+Bio&toStepper=true) .Then click on "continue with email"